specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchservice.json

```diff
@@ -1033,6 +1033,68 @@
         }
       }
     },
+    "/skillsets('{skillsetName}')/search.resetskills": {
+      "post": {
+        "tags": [
+          "Skillsets"
+        ],
+        "operationId": "Skillsets_ResetSkills",
+        "x-ms-examples": {
+          "SearchServiceResetSkills": {
+            "$ref": "./examples/SearchServiceResetSkills.json"
+          }
+        },
+        "description": "Reset an existing skillset in a search service.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/searchservice/preview-api/reset-skills"
+        },
+        "parameters": [
+          {
+            "name": "skillsetName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the skillset to reset."
+          },
+          {
+            "name": "skillNames",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "skillNames": {
```

Comment:
Why do we have a `skillNames` parameter that is an object with a `skillNames` property? Can we skip a level here?

Reply:
the Swagger ModelValidation doesn't allow me to. Looks like a body parameter must be an object and the actual parameters should be put in that object.

Reply:
Bummer.